### ISSUE TITLE
Fog::DNS::Google::Records example is broken

### ISSUE THREAD

**Issue by Temikus** [fog/fog-google#78]
get method in Fog::DNS::Google::Records supports only positional arguments

`connection.records.get` is structured like so:
```
def get(name, type)
  requires :zone

  records = service.list_resource_record_sets(zone.identity, { :name => name, :type => type }).body['rrsets'] || []
  records.any? ? new(records.first) : nil
rescue Fog::Errors::NotFound
  nil
end
```
, however, our example lists:
```
record = connection.records(zone: zone).get(name: 'tessts.example.org.',type: 'A')
```
, which leads to argument errors:
```
> record = connection.records(zone: zone).get(name: 'tessts.example.org.',type: 'A')
ArgumentError: wrong number of arguments (1 for 2)
```

Should I:
 - Fix the example?  
or
 - Fix the method to accept an options hash? ( Ruby 2.0 doesn't support _required_ named arguments :disappointed: )

/CC @plribeiro3000 @icco

**Comment by icco** [comment]
Please fix the example. Thanks for catching!

**Comment by Temikus** [comment]
Fixed by #81

**Comment by icco** [comment]
<3 ty.
